src/java/htsjdk/variant/vcf/VCFConstants.java

```diff
@@ -93,6 +94,9 @@
     public static final char DELETION_ALLELE_v3 = 'D';
     public static final char INSERTION_ALLELE_v3 = 'I';
 
+    // spanning deletion allele
+    public static final String SPANNING_DELETION_ALLELE = "*";
+
```

Comment:
I think this is better defined as a char instead of a String (since you'll need it for the wouldBeStarAllele() test).